internal/db/query.go

```diff
@@ -315,3 +368,32 @@ func (p *PackManager) IsPathPacked(path string) *string {
 
 	return nil
 }
+
+func CreateCache(ctx context.Context, tx pgx.Tx, prefix string) (int64, error) {
+	prefix = filepath.Join(prefix, "%")
+	sql := `
+		WITH impactful_packed_objects AS (
+			SELECT hash, count(*) as count
+			FROM dl.objects
+			WHERE path LIKE $1
+				AND packed = true
+				AND stop_version IS NULL
+			GROUP BY hash
+			ORDER BY count DESC
+			LIMIT 100
```

Comment:
Should we make this limit value an argument to this function?

Feels like the kind of thing we'll be tuning a lot and may even make an arg to the create cache RPC.